core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/controller/promisehandlers/WorkerStateUpdatedHandler.scala

```diff
@@ -30,7 +30,7 @@ trait WorkerStateUpdatedHandler {
         )
       sendToClient(
         ExecutionStatsUpdate(
-          cp.workflowExecution.getRunningRegionExecutions.flatMap(_.getStats).toMap
+          cp.workflowExecution.getAllRegionExecutionsStats
```

Comment:
also, I wonder the reason we change it from getRunningRegionExecutions to getAllRegionExecutions`.

Reply:
https://github.com/Texera/texera/pull/2570#discussion_r1554849374